.github/workflows/release.yaml

```diff
@@ -67,3 +67,18 @@ jobs:
       image_tags: ${{ github.event.release.tag_name }}
       git_ref: ${{ github.ref }}
     secrets: inherit
+
+  cloud-provider-plguins:
```

Comment:
`plugins` typo

Reply:
This was deleted in a later commit, so should be squashed for clarity

Reply:
squashed